### PR TITLE
Update onnx-tensorrt parser and cgmanifest

### PR DESCRIPTION
**Pull request by stevenlix**
1. Update onnx-tensorrt parser to the latest
2. Update cgmanifest.json

### CODE REVIEW

File: cgmanifests/submodules/cgmanifest.json [hunk 1]
```diff
@@ -335,52 +335,12 @@
...
         "type": "git",
         "git": {
-          "commitHash": "99296a4462ffc0a8a4a9635e51382277bec68979",
+          "commitHash": "53ed9b36050bc48e4e6fcc1996fb51c408c066b5",
           "repositoryUrl": "https://github.com/onnx/onnx-tensorrt.git"
         },
         "comments": "git submodule at cmake/external/onnx-tensorrt"
       }
     },
-    {
```
Comment (jywu-msft) [inline cgmanifests/submodules/cgmanifest.json:344]: are those supposed to be removed?
seems like it no longer recurses into submodules? just wanted to check if this is expected. @snnn

Comment (snnn) [reply cgmanifests/submodules/cgmanifest.json:344]: No. I guess Steven  didn't run 
```
git submodule update --init --recursive
```

Comment (stevenlix) [reply cgmanifests/submodules/cgmanifest.json:344]: I do run the git update

Comment (stevenlix) [reply cgmanifests/submodules/cgmanifest.json]: It turns out I need to run the script from root dir. Has rerun it and fixed the issue

**Review by jywu-msft — APPROVED** [review]

**Review by snnn — APPROVED** [review]

**Pull request merged**